tests/test_client.py

```diff
@@ -53,12 +53,12 @@
 }
 
 
-class TestDevnetClient(TestCase):
+class Testnet(TestCase):
```

Comment:
could be run with a option flag as so;

https://stackoverflow.com/questions/47559524/pytest-how-to-skip-tests-unless-you-declare-an-option-flag

Means we can still do dev work on dev and keep both classes to check both envs.

Reply:
how about this solution? 1cc681b46ccad045f43612272d491b40897ab4b3

`python -m pytest .` only tests on Testnet (what most people will run)
`python -m pytest . -m dev` tests on Devnet and should be more clear to people to expect breakage

also added `self.environment = Environment.DEVNET` to actually set a different environment in the second class